File: eSalon/eSalon.Model/RecenzijaOdgovor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class RecenzijaOdgovor
    {
        public int RecenzijaOdgovorId { get; set; }

        public int RecenzijaId { get; set; }

        public int KorisnikId { get; set; }

        public string Komentar { get; set; } = null!;

        public DateTime DatumDodavanja { get; set; }

        public int BrojLajkova { get; set; }

        public int BrojDislajkova { get; set; }

        public string? KorisnickoIme { get; set; }
        public string? KomentarRecenzije { get; set; }
    }
}

File: eSalon/UI/esalon_desktop/lib/models/recenzija_odgovor.dart
import 'package:json_annotation/json_annotation.dart';

part 'recenzija_odgovor.g.dart';

@JsonSerializable()
class RecenzijaOdgovor {
  int? recenzijaOdgovorId;
  int recenzijaId;
  int korisnikId;
  String komentar;
  DateTime datumDodavanja;
  int brojLajkova;
  int brojDislajkova;
  String? korisnickoIme;
  String? komentarRecenzije;
  String? nazivUsluge;

  RecenzijaOdgovor(
    this.recenzijaOdgovorId,
    this.recenzijaId,
    this.korisnikId,
    this.komentar,
    this.datumDodavanja,
    this.brojLajkova,
    this.brojDislajkova,
    this.korisnickoIme,
    this.komentarRecenzije,
    this.nazivUsluge,
  );

  factory RecenzijaOdgovor.fromJson(Map<String, dynamic> json) => _$RecenzijaOdgovorFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijaOdgovorToJson(this);
}